app/src/main/java/com/example/socialapp/models/Post.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.socialapp.models
+import com.google.gson.annotations.SerializedName
+class Post {
+    @SerializedName("name")
+    var name: String? = null
+
+    @SerializedName("id")
+    var id: String? = null
+    override fun toString(): String {
```

Comment:
data class will not have toString() method

Reply:
Changes done.